### PR TITLE
Task 5: Filter By Genre

### PR DESCRIPTION
**Pull request by kely-rufino**
### Description

In this PR I'm adding a filter by a selected genre on the loaded movies.

### Demo
task 5

### CODE REVIEW

**Comment by Kaylalo** [comment]
It works

**Comment by JoeLam930** [comment]
Wow! awesome

**Comment by luciavelasco** [comment]
To update this branch to include `Task 1` code:
`git pull origin JoeLam`
make sure you're on THIS branch
`git merge JoeLam`
Then wait for task 1 to be merged before Lucia can merge this.

File: src/components/meetups/FilterGenre.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+function FilterGenre({ onChange, selectedGenre }) {
+  const allGenres = [
```
Comment (luciavelasco) [inline src/components/meetups/FilterGenre.jsx:2]: Nice work! How did you get the list?

**Comment by dora-kovacs** [comment]
Really cool Kely, amazing job 🥇

File: src/components/meetups/FilterGenre.jsx (hunk 1 above)
Comment (kely-rufino) [reply src/components/meetups/FilterGenre.jsx:2]: Thank you!
 I did on the console of the browser. I take the JSON file from the Code Editor to console and there I use map and forEach to take all the genres. 

x.map((item) => item.genre).flat().forEach((genre) => allGenres[genre] = genre) 
JSON.stringify(Object.keys(allGenres))

And the magic happened :D

**Pull request merged**